apps/framework-cli/src/cli/routines.rs

```diff
@@ -313,13 +319,17 @@ pub async fn start_development_mode(
         )
         .await?;
 
+        let infra_map: &'static RwLock<InfrastructureMap> =
```

Comment:
I had not thought about - we need to keep this in memory. 

That works although I am not a big fan in that state being kept here. Not sure why though.